### PR TITLE
Fixes xwalk not ready crash in onNewIntent

### PR DESCRIPTION
**Pull request by steam**
Many functions on `XWalkView` are not callable until `onXWalkReady()`. We were calling `onNewIntent()` too early.

[fixes #125966995](https://www.pivotaltracker.com/story/show/125966995)

### CODE REVIEW

File: app/src/main/java/co/ello/ElloApp/MainActivity.java [hunk 1]
```diff
@@ -168,7 +168,9 @@ protected void onNewIntent(Intent intent) {
             isDeepLink = true;
         }
 
-        xWalkView.onNewIntent(intent);
+        if (isXWalkReady) {
+            xWalkView.onNewIntent(intent);
+        }
```
Comment (jayzes) [inline app/src/main/java/co/ello/ElloApp/MainActivity.java]: Do we need to defer this or something so it gets called later when Crosswalk is ready? Or is it okay to just let it drop?

Comment (steam) [inline app/src/main/java/co/ello/ElloApp/MainActivity.java]: I **_think_** we'll be fine. We'll have to test a bit. We do this a lot in here and it has never hurt anything as far as I can tell.

Comment (jayzes) [inline app/src/main/java/co/ello/ElloApp/MainActivity.java]: 👍 

I don't actually know anything about the underlying frameworks, so I compensate by raising issues.

Comment (steam) [inline app/src/main/java/co/ello/ElloApp/MainActivity.java]: I do wonder it's impact as well.

**Pull request merged**